### PR TITLE
add toggle function

### PR DESCRIPTION
**Pull request by Sukhpreet92**

### CODE REVIEW

File: components/Card.js [hunk 1]
```diff
@@ -2,9 +2,15 @@ import styled from "styled-components";
...
   return (
-    <CardContainer>
+    <CardContainer isChecked={isChecked}>
+      <StyledInput
+        id={id}
+        type="radio"
+        defaultChecked={isChecked}
+        onClick={onToggle}
+      />
       {name}
```
Comment (AHen-C0de) [inline components/Card.js]: put the name in a `<label>` element and connect it with the input via `htmlFor`.. then you can also toggle your habit by clicking the name, not only the checkbox ;)

File: pages/index.js [hunk 2]
```diff
@@ -34,23 +41,62 @@ export default function Home() {
...
+                  <Card
+                    key={card.id}
+                    name={card.name}
+                    onDelete={() => deleteCard(card.id)}
+                    onToggle={() => toggleCard(card.id)}
+                  />
+                );
+              })}
+          </HabitForTodayList>
+          <br />
```
Comment (AHen-C0de) [inline pages/index.js]: Why do you need a break element here? If it is for styling reasons, better solve it with CSS grid or flexbox.

File: pages/index.js [hunk 3]
```diff
@@ -34,23 +41,62 @@ export default function Home() {
...
+    </Wrap>
   );
 }
 
 const Main = styled.main`
-  background-color: white;
+  margin: 0;
+`;
+
+const Wrap = styled.div`
```
Comment (AHen-C0de) [inline pages/index.js]: it looks unusual to have a div around the Header and the Main. Could you transfer these settings to the GlobalStyle.js?

**Review by AHen-C0de — CHANGES_REQUESTED** [review]
Hey,
looks very good! Just added some minor suggestions. ;)

File: components/Card.js (hunk 1 above)
Comment (Sukhpreet92) [reply components/Card.js]: Thank you for your great suggestion. i inserted the <labe>

File: pages/index.js (hunk 2 above)
Comment (Sukhpreet92) [reply pages/index.js]: I have removed the <br />

File: pages/index.js (hunk 3 above)
Comment (Sukhpreet92) [reply pages/index.js]: Hey thank you @AHen-C0de  i was playing with the background image but forgot to remove the div.

**Review by AHen-C0de — APPROVED** [review]
LGTM!

**Pull request merged**